### ISSUE TITLE
[Magento2] Find usages of the interface method produces an exception

### ISSUE THREAD

**Issue by Koc** [enriquepiatti/Magicento#247]
```
Argument for @NotNull parameter 'parent' of com/magicento2/helpers/PsiPhpHelper.getAllMethodReferences must not be null
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'parent' of com/magicento2/helpers/PsiPhpHelper.getAllMethodReferences must not be null
	at com.magicento2.helpers.PsiPhpHelper.getAllMethodReferences(PsiPhpHelper.java)
	at com.magicento2.models.phpclass.Model.getInitMethodCall(Model.java:47)
	at com.magicento2.models.phpclass.Model.getResourceModelName(Model.java:55)
	at com.magicento2.extensions.type.PhpTypeProvider2.getResourceClassFromSignature(PhpTypeProvider2.java:236)
	at com.magicento2.extensions.type.PhpTypeProvider2.getBySignature(PhpTypeProvider2.java:274)
	at com.jetbrains.php.PhpIndexImpl.getBySignatureInternal(PhpIndexImpl.java:482)
	at com.jetbrains.php.PhpIndexImpl.getBySignature(PhpIndexImpl.java:326)
	at com.jetbrains.php.PhpIndexImpl.getClasses(PhpIndexImpl.java:514)
	at com.jetbrains.php.PhpIndexImpl.getBySignatureInternal(PhpIndexImpl.java:450)
	at com.jetbrains.php.PhpIndexImpl.getBySignature(PhpIndexImpl.java:326)
	at com.jetbrains.php.PhpIndexImpl.getClasses(PhpIndexImpl.java:514)
	at com.jetbrains.php.PhpIndexImpl.getBySignatureInternal(PhpIndexImpl.java:450)
	at com.jetbrains.php.PhpIndexImpl.getBySignature(PhpIndexImpl.java:326)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:81)
	at com.jetbrains.php.lang.psi.resolve.types.PhpType.globalLocationAware(PhpType.java:417)
	at com.jetbrains.php.lang.psi.elements.impl.MemberReferenceImpl.resolveGlobal(MemberReferenceImpl.java:140)
	at com.jetbrains.php.lang.psi.elements.impl.MethodReferenceImpl.resolveGlobal(MethodReferenceImpl.java:84)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl$1.resolve(PhpReferenceImpl.java:145)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl$1.resolve(PhpReferenceImpl.java:142)
	at com.intellij.psi.impl.source.resolve.ResolveCache.lambda$resolve$0(ResolveCache.java:140)
	at com.intellij.openapi.util.RecursionManager$2.doPreventingRecursion(RecursionManager.java:112)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolve(ResolveCache.java:139)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:163)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:155)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl.multiResolve(PhpReferenceImpl.java:137)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl.resolve(PhpReferenceImpl.java:123)
	at com.jetbrains.php.lang.psi.elements.impl.MemberReferenceImpl.isReferenceTo(MemberReferenceImpl.java:233)
	at com.intellij.psi.search.SingleTargetRequestResultProcessor.processTextOccurrence(SingleTargetRequestResultProcessor.java:52)
	at com.intellij.psi.impl.search.PsiSearchHelperImpl$3.lambda$execute$0(PsiSearchHelperImpl.java:754)
	at com.intellij.psi.impl.search.LowLevelSearchUtil.processTreeUp(LowLevelSearchUtil.java:138)
	at com.intellij.psi.impl.search.LowLevelSearchUtil.processElementsAtOffsets(LowLevelSearchUtil.java:224)
	at com.intellij.psi.impl.search.PsiSearchHelperImpl$3.execute(PsiSearchHelperImpl.java:750)
	at com.intellij.psi.impl.search.PsiSearchHelperImpl$2.processInReadAction(PsiSearchHelperImpl.java:247)
	at com.intellij.psi.impl.search.PsiSearchHelperImpl$2.processInReadAction(PsiSearchHelperImpl.java:238)
	at com.intellij.openapi.application.ReadActionProcessor.lambda$process$0(ReadActionProcessor.java:29)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:942)
	at com.intellij.openapi.application.ReadActionProcessor.process(ReadActionProcessor.java:29)
	at com.intellij.psi.impl.search.PsiSearchHelperImpl.lambda$processCandidates$13(PsiSearchHelperImpl.java:712)
	at com.intellij.psi.impl.search.PsiSearchHelperImpl.lambda$processVirtualFile$5(PsiSearchHelperImpl.java:421)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1153)
	at com.intellij.openapi.application.ex.ApplicationUtil.tryRunReadAction(ApplicationUtil.java:39)
	at com.intellij.psi.impl.search.PsiSearchHelperImpl.processVirtualFile(PsiSearchHelperImpl.java:405)
	at com.intellij.psi.impl.search.PsiSearchHelperImpl.lambda$processPsiFileRoots$2(PsiSearchHelperImpl.java:323)
	at com.intellij.psi.impl.search.PsiSearchHelperImpl.lambda$processFilesConcurrentlyDespiteWriteActions$3(PsiSearchHelperImpl.java:351)
	at com.intellij.concurrency.ApplierCompleter.d(ApplierCompleter.java:133)
	at com.intellij.concurrency.ApplierCompleter.b(ApplierCompleter.java:222)
	at com.intellij.concurrency.ApplierCompleter.d(ApplierCompleter.java:151)
	at com.intellij.concurrency.ApplierCompleter.b(ApplierCompleter.java:222)
	at com.intellij.concurrency.ApplierCompleter.d(ApplierCompleter.java:151)
	at com.intellij.concurrency.ApplierCompleter.b(ApplierCompleter.java:222)
	at com.intellij.concurrency.ApplierCompleter.d(ApplierCompleter.java:151)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:556)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:501)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:66)
	at com.intellij.concurrency.ApplierCompleter.a(ApplierCompleter.java:116)
	at com.intellij.concurrency.ApplierCompleter.c(ApplierCompleter.java:96)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:142)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:242)
	at com.intellij.concurrency.ApplierCompleter.compute(ApplierCompleter.java:96)
	at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
```

**Comment by Koc** [comment]
got this error again
```
Argument for @NotNull parameter 'parent' of com/magicento2/helpers/PsiPhpHelper.getAllMethodReferences must not be null
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'parent' of com/magicento2/helpers/PsiPhpHelper.getAllMethodReferences must not be null
	at com.magicento2.helpers.PsiPhpHelper.getAllMethodReferences(PsiPhpHelper.java)
	at com.magicento2.models.phpclass.Model.getInitMethodCall(Model.java:47)
	at com.magicento2.models.phpclass.Model.getResourceModelName(Model.java:55)
	at com.magicento2.extensions.type.PhpTypeProvider2.getResourceClassFromSignature(PhpTypeProvider2.java:236)
	at com.magicento2.extensions.type.PhpTypeProvider2.getBySignature(PhpTypeProvider2.java:274)
	at com.jetbrains.php.PhpIndexImpl.getBySignatureInternal(PhpIndexImpl.java:482)
	at com.jetbrains.php.PhpIndexImpl.getBySignature(PhpIndexImpl.java:326)
	at com.jetbrains.php.PhpIndexImpl.getClasses(PhpIndexImpl.java:514)
	at com.jetbrains.php.PhpIndexImpl.getBySignatureInternal(PhpIndexImpl.java:450)
	at com.jetbrains.php.PhpIndexImpl.getBySignature(PhpIndexImpl.java:326)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:81)
	at com.jetbrains.php.lang.psi.resolve.types.PhpType.globalLocationAware(PhpType.java:417)
	at com.jetbrains.php.lang.psi.elements.impl.MemberReferenceImpl.resolveGlobal(MemberReferenceImpl.java:140)
	at com.jetbrains.php.lang.psi.elements.impl.MethodReferenceImpl.resolveGlobal(MethodReferenceImpl.java:84)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl$1.resolve(PhpReferenceImpl.java:145)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl$1.resolve(PhpReferenceImpl.java:142)
	at com.intellij.psi.impl.source.resolve.ResolveCache.lambda$resolve$0(ResolveCache.java:140)
	at com.intellij.openapi.util.RecursionManager$2.doPreventingRecursion(RecursionManager.java:112)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolve(ResolveCache.java:139)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:163)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:155)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl.multiResolve(PhpReferenceImpl.java:137)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl.resolve(PhpReferenceImpl.java:123)
	at com.intellij.codeInsight.TargetElementUtil.b(TargetElementUtil.java:391)
	at com.intellij.codeInsight.TargetElementUtil.a(TargetElementUtil.java:368)
	at com.intellij.codeInsight.TargetElementUtil.a(TargetElementUtil.java:238)
	at com.intellij.codeInsight.TargetElementUtil.findTargetElement(TargetElementUtil.java:210)
	at com.intellij.codeInsight.daemon.impl.IdentifierHighlighterPass.doCollectInformation(IdentifierHighlighterPass.java:92)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:70)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.c(PassExecutorService.java:438)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1153)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.b(PassExecutorService.java:431)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:556)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:501)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:66)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.a(PassExecutorService.java:430)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.d(PassExecutorService.java:406)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:142)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:242)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:404)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:165)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
```

**Comment by enriquepiatti** [comment]
fixed in 1.6.0

**Comment by Koc** [comment]
good, will check
